.infrastructure/deployment.yml

```diff
@@ -45,8 +45,27 @@ spec:
         - name: SECRET_KEY
           valueFrom:
             secretKeyRef:
-              name: app-secret
+              name: mysql-secrets
               key: SECRET_KEY
+        - name: DB_HOST
+          value: "mysql-service"
+        - name: DB_PORT
+          value: "3306"
+        - name: DB_USER
+          valueFrom:
+            secretKeyRef:
+              name: mysql-secrets
+              key: MYSQL_USER
+        - name: DB_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: mysql-secrets
```

Comment:
The secret name `mysql-secrets` used here does not match the secret name `app-secret` defined in the volumes section. Ensure consistency in secret names to avoid runtime errors.